tools/docker/builder/openwrt/scripts/build-openwrt.sh

```diff
@@ -34,6 +35,23 @@ case $TARGET_DEVICE in
             echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}=y"
             echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y"
             echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y"
+            # AMBIORIX modules to work with u-bus
+            echo "CONFIG_PACKAGE_amxb-ubus=y"
+            # AMBIORIX applications for debug
+            echo "CONFIG_PACKAGE_amxb-inspect=y"
+            echo "CONFIG_PACKAGE_amxo-cg=y"
+            echo "CONFIG_PACKAGE_amxrt=y"
+            # AMBIORIX libs
+            echo "CONFIG_PACKAGE_libamxb=y"
+            echo "CONFIG_PACKAGE_libamxc=y"
```

Comment:
The same changes have to be introduced in prplwrt for the axepoint/rax40 in a profile file.
This one is not a nitpick comment, if we want to use it on the axepoint it has to be in a profile under `profiles/`, since `gen_config.py` removes any custom `.config` line anyway.